docs/pages/blog/2020-developer-survey-results.md

```diff
@@ -179,10 +179,38 @@ As the answers to this questions were pretty different, we grouped them into dif
   <tr><th>3</th><th> grid - improve</th><tr>
 </table>
 
+### Comparison with last year
+
+There are a couple of noticeable differences compared to last year.
+Some can be explained by our work, others by an evolution of the ecosystem.
+Each item is prefixed by the multiplicator factor of pain point between 2019 and 2020.
+
+Decreasing pain:
+- x0: Slider. No requests, the problem is almost solved since the introduction of the range feature.
+- x0.1: Strict mode. We did a lot for it this year. However, since Create React App has made this mode a default, we have seen a lot more requests for it.
+- x0.1: Autocomplete. We did a lot for it this year. The component will be moved to the core in v5.
+- x0.2: Fewer breaking changes. Only releasing minor versions under v4 for over a year helps a lot. However, we still need to be careful with CSS changes.
+- x0.3: Accessibility. We did a lot for it this year. One of our best leverage has been GitHub's issue opened by a11y expert, often coming from large companies using Material-UI at scale.
+- x0.4: Material design. We didn't do much for it this year, at least not as much as we could have. Maybe fewer people care? It seems that we resonate better with developers going for custom design systems.
+- x0.4: TypeScript. Our effort is paying off with the migration of all the demos to TypeScript and the migration of all the props to IntelliSense.
+- x0.5: Date picker. We did a lot for it this year. We probably still need the range feature and the integration in the main repository (docs migration, etc.) for consistency to get is much lower, to the level of the "Slider".
+- x0.7: Performance. We didn't do much this year, maybe more developers are used to leverage React effectively? Virtualization updates pruning, etc.
+
+Growing pain:
+- ∞: Form is a new item. It seems that we should at least work more closely with react-hook-form, formik, and react-final-form.
+- ∞: Charts is a new item. Material Design even has a page dedicated to [date visualization](https://material.io/design/communication/data-visualization.html) since recently.
+- x5: Custom themes.
+- x5: Simpler customization. We have improved customizability this year by introducing global class names and reducing the CSS specificity of some selectors. However, it seems that we are now tapping into a new audience. We need to do better.
+- x1.5: Animations.
+- x1.2: React native. We still have no plans for it. The [market is too small](https://npm-stat.com/charts.html?package=react-dom,react-native) to make it sustainable with our model.
+- x1.1: More components, the more we give, the more developer asks for, we will fully solve this with the enterprise version. The first early access will land this year.
```

Comment:
> we will fully solve this

That isn't true. There will always be a long tail of niche components that we don't provide.

> with the enterprise version

And that will be the opposite - a limited set of advanced components.

Reply:
```suggestion
- x1.1: More components, the more we give, the more developer asks for. We will try to better solve this with the enterprise version. Both because it's the only model we have found that can sustain the developments of data-rich advanced components and because it allows us to invest back resources into the MIT side. The first early access will land this year.
```